packages/frontend/index.d.ts

```diff
@@ -4,7 +4,7 @@ interface ArticleProps {
     config: ConfigType;
 }
 
-type Pillar = 'news' | 'opinion' | 'sport' | 'culture' | 'lifestyle';
+type Pillar = 'news' | 'opinion' | 'sport' | 'culture' | 'lifestyle' | 'labs';
```

Comment:
Labs seems intuitive enough! Does it match with language we use elsewhere? I.e. do you know if design will understand this/use this term? (I don't know the answer but suspect this is okay.)